proto/buf/alpha/registry/v1alpha1/studio_request.proto

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+// StudioRequestService exposes the functionality to manage favorited Requests
+// from Studio.
+service StudioRequestService {
+  // Create registers a favorite Studio requests to the caller's BSR profile.
+  rpc Create(CreateRequest) returns (CreateResponse) {}
+  // Delete removes a favorite Studio request from the caller's BSR profile.
+  rpc Delete(DeleteRequest) returns (DeleteResponse) {}
+  // List shows the caller's favorited requests.
+  rpc List(ListRequest) returns (ListResponse) {}
+}
+
+// CreateRequest is the proto request representation of the Create method. IDs
+// and timestamps are automatically generated, BSR Repository information is
+// taken from the `repository_id`, and the `author` is taken from the request
+// authentication.
+message CreateRequest {
+  // BSR repository ID this request is linked to.
+  string repository_id = 1;
+  // The friendly name of this request.
+  string name = 2;
+  // The target base URL of this request.
+  string target_base_url = 3;
+  // The target service name as defined in the BSR repository.
+  string service = 6;
+  // The target RPC method name as defined in the BSR repository.
+  string method = 7;
+  // The request body as defined in Studio.
+  string request_body = 8;
+  // The request headers as defined in Studio.
+  map<string, string> request_headers = 9;
+  // The response body as seen in Studio.
+  string response_body = 10;
+  // The response headers as seen in Studio.
+  map<string, string> response_headers = 11;
+}
+
+// CreateResponse is the proto response representation of the Create method.
+message CreateResponse {
```

Comment:
This is a global name `CreateResponse` in the main registry package, this should be scoped as `CreateStudioRequestResponse` or the like

Reply:
You're right, fixed.